app/controllers/curation_concerns/generic_works_controller.rb

```diff
@@ -32,6 +30,19 @@ def deny_access
            locals: { presenter: presenter }
   end
 
+  def relationships
+    id = params[:id]
+    query = "preferred_representation_ssim:#{id}"
+    params = { rows: 100, fl: ["id", "pref_label_tesim"] }
+    solr_hits = ActiveFedora::SolrService.query(query, params).map do |solr_hit|
```

Comment:
When there's time, we should move this new endpoint hash manipulation to a jbuilder like we did the autocomplete's index.